### PR TITLE
Delete Command and Portfolio Page

### PR DESCRIPTION
**Pull request by nihalzp**
Tests and other bugs fixes coming in next PR.

### CODE REVIEW

File: src/main/java/athleticli/parser/Parser.java [hunk 1]
```diff
@@ -120,6 +124,8 @@ public static Command parseCommand(String rawUserInput) throws AthletiException
             return new FindActivityCommand(parseDate(commandArgs));
         case CommandName.COMMAND_ACTIVITY_GOAL_SET:
             return new SetActivityGoalCommand(ActivityParser.parseActivityGoal(commandArgs));
+        case CommandName.COMMAND_ACTIVITY_GOAL_DELETE:
+            return new DeleteActivityGoalCommand(ActivityParser.parseDeleteActivityGoal(commandArgs));
```
Comment (DaDevChia) [inline src/main/java/athleticli/parser/Parser.java:128]: Good feature, but lets not introduce any more new commands and features after the feature freeze, as that would create more testing required.

File: src/main/java/athleticli/parser/Parser.java [hunk 2]
```diff
@@ -173,4 +179,33 @@ public static LocalDate parseDate(String date) throws AthletiException {
...
+        String patternString = "";
+
+        if (marker.equals(Parameter.DATETIME_SEPARATOR)) {
+            // Special handling for datetime to capture the date and time
+            patternString = marker + "(\\d{4}-\\d{2}-\\d{2} \\d{2}:\\d{2})";
+        } else {
+            // For other markers, capture a sequence of non-whitespace characters
+            patternString = marker + "(\\S+)";
+        }
+
```
Comment (DaDevChia) [inline src/main/java/athleticli/parser/Parser.java:199]: Good to have a rejex checker to capture the date and time!

**Review by DaDevChia — APPROVED** [review]
Looks good to me, but lets make this the final feature addition.

File: src/main/java/athleticli/parser/Parser.java (hunk 1 above)
Comment (nihalzp) [reply src/main/java/athleticli/parser/Parser.java:128]: Yes good idea and this feature was proposed and on the bucket list last week. I only got the chance to implement it now.

**Pull request merged**